### ISSUE TITLE
Slipping strips not mentioned

### ISSUE THREAD

**Issue by afranke** [blendoc/vse-docs#7]
I was looking for how to slip strip content and couldn’t find it. I didn’t have the proper term either, so it took some searching, but I eventually found it in the official manual. I think this would be worth adding to this documentation.

**Comment by ghost** [comment]
Yes, it would. I will try to add it. But, lately, the project is a little bit on hold because of the difficulties to integrate it with the official Blander docs.
If you should be interested in participating with these unofficial docs, please give me a call.
